new_files/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/StdModule.php

```diff
@@ -252,7 +252,7 @@ public function addConfigurationDropDownByStaticFunction($key, $value, $groupId,
         $this->addConfiguration($key, $value, $groupId, $sortOrder, $setFunction);
     }
 
-    public function addConfiguration($key, $value, $groupId, $sortOrder, $setFunction = '', $useFunction = '')
+    public function addConfiguration(string $key, string $value = '', int $groupId = 6, int $sortOrder = 1, string $setFunction = '', string $useFunction = '')
```

Comment:
LGTM but we do not use type hints in the std module. Although we are not uniform on this either. In a later version of the std module, we can equip the entire module with type hints.

Reply:
Alrighty! I deliberately didn't update from `main`, is it necessary?